api/src/pcapi/alembic/versions/20240531T073532_2e66d0e6d190_.py

```diff
@@ -0,0 +1,28 @@
+"""Add OffererAddress link to CollectiveOfferTemplate"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "2e66d0e6d190"
+down_revision = "0fdfda2e6800"
+branch_labels: tuple[str] | None = None
+depends_on: list[str] | None = None
+
+
+def upgrade() -> None:
+    op.add_column("collective_offer_template", sa.Column("offererAddressId", sa.BigInteger(), nullable=True))
+    op.create_foreign_key(
+        "collective_offer_templace_offererAddressId_fkey",
+        "collective_offer_template",
+        "offerer_address",
+        ["offererAddressId"],
+        ["id"],
+        postgresql_not_valid=True,
```

Comment:
Dans ce cas là si il y a un echec sur la création de la FKey, le retry aura lui une erreur sur le add_column et la migration ne passera pas. 
Tu peux modifier le add_column pour utiliser un IF NOT EXISTS [comme ici](https://github.com/pass-culture/pass-culture-main/pull/12480/files#diff-330445586eda71d70d98fcc188cbc33f541a10294f25d1345d454e18908781b4). Tu pourra faire la création de la colonne + la foreign key sur la même migration